dmoj/cptbox/_cptbox.pyx

```diff
@@ -445,13 +443,7 @@ cdef class Process:
             config.stderr_ = self._child_stderr
             config.argv = alloc_byte_array(args)
             config.envp = alloc_byte_array(env)
-            if fds is None or not len(fds):
-                config.max_fd = 2
-            else:
-                config.max_fd = 2 + len(fds)
-                config.fds = <int*>malloc(sizeof(int) * len(fds))
-                for i in range(len(fds)):
-                    config.fds[i] = fds[i]
+            config.max_fd = 2
```

Comment:
`max_fd` is never not 2, let's inline it

Reply:
Done.